maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/RepositoryManagerDriver.java

```diff
@@ -270,14 +280,39 @@ private void setupBuildRepos(BuildExecution execution, Indy indy)
             // Global-level repos, for captured/shared artifacts and access to the outside world
             addGlobalConstituents(buildGroup, tempBuild);
 
-            // add extra repositories removed from poms by the adjust process
-            addExtraConstituents(execution.getArtifactRepositories(), id, buildContentId, indy, buildGroup);
+            // add extra repositories removed from poms by the adjust process and set in BC by user
+            List<ArtifactRepository> extraDependencyRepositories = extractExtraRepositoriesFromGenericParameters(genericParameters);
+            if(execution.getArtifactRepositories() != null) {
```

Comment:
Missing space after "if"

Reply:
Ah, I keep forgetting about it. I was used to do it without space. Updated :-)